requirements.txt

```diff
@@ -9,3 +9,6 @@ grpcio-tools>1.34.0,<1.42
 grpcio>1.34.0,<1.42
 v3io-frames>=0.10.2,<1
 v3iofs~=0.1.9
+redis~=3.5
+redis-py-cluster~=2.1.3
+lupa~=1.13
```

Comment:
These should probably be in `extras_require` (see `setup.py`).

Reply:
What is the guide to separate requirements.txt and extras_require()?

Reply:
We discussed the difference between ordinary requirements and extra requirements (the latter are optional).

Reply:
done